packages/core/stories/accordion/accordion.stories.tsx

```diff
@@ -186,3 +188,38 @@ export const Status: StoryFn<AccordionGroupProps> = (props) => (
     ))}
   </AccordionGroup>
 );
+
+export const AdditionalLabels: StoryFn<AccordionProps> = (props) => (
+  <Accordion style={{ width: "90%" }} {...props}>
+    <AccordionHeader>
+      <StackLayout direction={"row"} style={{ width: "100%" }}>
+        <StackLayout gap={0.5} align="start">
+          <Text>Title</Text>
+          <Text color="secondary">Description goes here</Text>
+        </StackLayout>
+        <div style={{ marginLeft: "auto" }}>
```

Comment:
This feels like a https://www.saltdesignsystem.com/salt/components/split-layout/examples

Reply:
added splitlayout